packages/flutter/lib/src/foundation/synchronous_future.dart

```diff
@@ -4,6 +4,8 @@
 
 import 'dart:async';
 
+export 'dart:async' show FutureOr;
```

Comment:
ugh, no getting away from this one i guess.

i would be tempted to suggest an `//ignore` on this one and just export Future...

Reply:
exporting `Future` is a noop because it's already exported by `dart:core`. `FutureOr` is a littlle special because most of the time users don't use directly `FutureOr` they use `Future` or a type. So I think it makes sense to special case this type.